resources/cassandra-env-sh/cassandra/cassandra-env-sh-cassandra-4.0.0.template

```diff
@@ -120,7 +120,7 @@ echo $JVM_OPTS | grep -q Xms
 DEFINED_XMS=$?
 echo $JVM_OPTS | grep -q UseConcMarkSweepGC
 USING_CMS=$?
-echo $JVM_OPTS | grep -q UseG1GC
+echo $JVM_OPTS | grep -q +UseG1GC
```

Comment:
is this plus upstream now?

Reply:
https://github.com/apache/cassandra/blob/trunk/conf/cassandra-env.sh#L123

Reply:
Err, I thought that would inline the code. Yes, it changed upstream.